draft-ietf-acme-acme.md

```diff
@@ -937,11 +938,13 @@ not recognize.  In particular, it MUST NOT reflect unrecognized fields in the
 resulting account object.  This allows clients to detect when servers do not
 support an extension field.
 
-The server SHOULD validate that the contact URLs in the "contact" field are
-valid and supported by the server.  If the client provides the server with an
-invalid or unsupported contact URL, then the server MUST return an error of type
-"invalidContact", with a description describing the error and what types of
-contact URL the server considers acceptable.
+The server SHOULD validate that the contact URIs in the "contact" field are
+valid and supported by the server. If the client provides the server with
+a contact URI using an unsupported protocol it MUST return an error of type
+"unsupportedContact", with a description describing the error and what types of
+contact URIs the server considers acceptable. The server MUST support the mailto
+scheme for contact URIs. If the client provides the server with an invalid
+contact URI, then the server MUST return an error of type "invalidContact".
```

Comment:
The changes look good, but in re-reading this I noticed a pre-existing issue.

It seems like validation described in the first sentence ("The server SHOULD validate that the contact URIs in the "contact" field are valid ...) would be required to adhere to the MUST from the last sentence ("If the client provides the server with an invalid contact URI, then the server MUST ..."). 

It seems like there are a few ways to address this. Two that come to mind would be:
(a) make the first sentence of this paragraph a MUST as well
(b) further qualify the two MUSTs so that they are mandatory only if the server implements validation (e.g., "If the server validates the contact URI and the client provides the server with an invalid contact URI, then ...")